core/src/navigation/LeftNav.svelte

```diff
@@ -496,8 +579,374 @@
   on:blur={closePopupMenu}
   on:keydown={closePopupMenuOnEsc}
 />
+{#if btpToolLayout}
+      <div 
+      class="fd-navigation fd-navigation--vertical {hideNavComponent
+        ? 'hideNavComponent'
+        : ''} {footerText || semiCollapsibleButton
+        ? 'hasFooter'
+        : ''} {footerText && !semiCollapsibleButton ? 'hasOnlyFooterText' : ''}
+        {isSemiCollapsed
+          ? 'fd-navigation--snapped'
+          : ''}" 
+      role="navigation" 
+      style="width: var(--luigi__left-sidenav--width); height: 100%;">
```

Comment:
This might be easier to understand:
```suggestion
  <div 
    class="fd-navigation fd-navigation--vertical" 
    class:hideNavComponent={hideNavComponent}
    class:hasFooter={footerText || semiCollapsibleButton}
    class:hasOnlyFooterText={footerText && !semiCollapsibleButton}
    class:fd-navigation--snapped={isSemiCollapsed}
    role="navigation" 
    style="width: var(--luigi__left-sidenav--width); height: 100%;">
```